pages/common/rg.md

```diff
@@ -31,3 +31,7 @@
 - Show lines that do not match the given pattern:
 
 `rg --invert-match {{pattern}}`
+
+- Search a literal string pattern:
+
+`rg -F {{string Choose which email address to assorg_to_search}}`
```

Comment:
```suggestion
`rg -F {{string}}`
```

Can you explain what is `string Choose which email address to assorg_to_search` supposed to mean? Just `string` seems much more clear.

Reply:
sorry accidental copy paste (something wrong with my editor). This should be fixed